config/routes.rb

```diff
@@ -14,7 +14,14 @@
     end
   end
 
+  resources :course_instances, only: [] do
+    collection do
+      get :autocomplete
+    end
+  end
```

Comment:
Since we aren't using any of the other resources, why not just do:
get 'course_instances/autocomplete' => 'course_instances#autocomplete'